SmartEnrollmentFor911/SmartEnrollmentFor911/Pages/Index.cshtml

```diff
@@ -13,6 +13,25 @@
 
 <p>Smart911 Enrollments details in the areas where crimes are reported.</p>
 
+<head>
+    <style type="text/css" media="screen">
```

Comment:
This is a good step.  Consider moving to a linked stylesheet so that this styling can be applied on other pages.